tests/components/vesync/test_humidifier.py

```diff
@@ -44,6 +45,7 @@ async def test_humidifier_state(
         assert hass.states.get(entity_id).state != STATE_UNAVAILABLE
 
     assert hass.states.get("sensor.humidifier_200s_humidity").state == "35"
+    assert hass.states.get("number.humidifier_200s_mist_level").state == "6"
```

Comment:
Oh lol, missed that last review. Let's test the sensor in the sensor tests and the number in the number tests